.github/workflows/linting.yml

```diff
@@ -70,10 +68,10 @@ jobs:
         with:
           python-version: ${{ env.PYTHON_VERSION }}
 
-      - name: Install Haystack and the dependencies needed for tests
-        run: pip install -r test/test_requirements.txt
+      - name: Install Hatch
+        run: pip install hatch
 
       - name: Pylint
         if: steps.files.outputs.any_changed == 'true'
         run: |
-          pylint -ry -j 0 ${{ steps.files.outputs.all_changed_files }}
+          hatch run test:pylint -ry -j 0 ${{ steps.files.outputs.all_changed_files }}
```

Comment:
what about ruff btw?

Reply:
Would be great to add that but not in this PR, it could be quite disruptive.

Reply:
yep, agree 👍🏻